tests/server/api/test_folders.py

```diff
@@ -0,0 +1,52 @@
+# coding=utf-8
+"""
+Unit tests for ``octoprint.server.api`` folders.
+"""
+
+from __future__ import absolute_import
+
+__author__ = "Gina Häußge <osd@foosel.net>"
+__license__ = 'GNU Affero General Public License http://www.gnu.org/licenses/agpl.html'
+__copyright__ = "Copyright (C) 2016 The OctoPrint Project - Released under terms of the AGPLv3 License"
+
+
+import json
+from mock import MagicMock
+import psutil
+from random import randint
```

Comment:
I'm not a huge fan of random test data in tests - can make it impossible to reproduce a failure.

Reply:
OK, no problem, we can use some static values.